app/views/landing.scala.html

```diff
@@ -15,41 +15,59 @@
  *@
 
 @import config.AppConfig
-@import repositories.declaration.Submission
+@import models.Declaration
+@import models.DeclarationActionType.DeclarationActionType
 @import uk.gov.hmrc.play.views.formatting.Dates._
 @import org.joda.time.LocalDate
 
-@(submissions: Seq[Submission])(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
+@(submissions: Seq[Declaration])(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
```

Comment:
Rename to variable to declarations
```suggestion
@(declarations: Seq[Declaration])(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
```